src/inputs/TextFieldBase.tsx

```diff
@@ -237,11 +245,23 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
               }}
               {...hoverProps}
               ref={inputWrapRef as any}
+              onClick={unfocusedPlaceholder ? handleUnfocusedPlaceholderClick : undefined}
```

Comment:
**thought:** Ahh, this is interesting. I think we actually have this "problem" currently. If a user clicks within the `inputWrap`, but not on the `input` then the field doesn't get focused. 🤔 I don't think its worth trying to address that here for all cases, but I see what you're doing.

Reply:
This also works the same on the `unfocusedPLaceholder` container. We can move it there if that makes more sense.

Reply:
Let's leave it as is. It hasn't been called out as an issue and I rather not introduce too many additional interaction handlers if we don't really need them.